seed/challenges/04-video-challenges/jslingo.json

```diff
@@ -63,7 +63,7 @@
       "description": [
         "We are going to cover what constitutes a variable, and the reasoning behind camelCase.",
         "A variable, also referred to as 'var', is the name or placeholder for a boolean, string, number, or other piece of static information.",
-        "You can use google dev tools to inspect the Free Code Camp home page and look for some variables.",
+        "You can use Google devtools to inspect the Free Code Camp home page and look for some variables.",
```

Comment:
Earlier these were referred to as `Dev Tools` so this should be changed for consistency.